dojo/tools/yarn_audit/parser.py

```diff
@@ -54,6 +54,18 @@ def get_item(item_node, test):
         if len(finding['paths']) > 25:
             paths += "\n  - ..... (list of paths truncated after 25 paths)"
 
+    cwe = item_node['cwe']
+    if cwe.startswith('CWE-'):
+        cwe = int(cwe[4:])
+    elif cwe.startswith('['):
+        # Somehow multiple CWEs end up in the json report like this
+        # [\"CWE-173\",\"CWE-200\",\"CWE-601\"]
+        # which becomes after json load:
+        # ["CWE-173","CWE-200","CWE-601"]
+        # we parse this and take the first CWE
+
+        cwe = json.loads(cwe)[0][4:]
```

Comment:
```suggestion
        cwe = json.loads(cwe)[0]
```
Should it be like this?

Reply:
No, cwe is an integer field.